installbuilder/Linux_RPM.data

```diff
@@ -0,0 +1,5 @@
+%Variables
+PACKAGE_TYPE: 'RPM'
+
+%Dependencies
+omi (>= 1.3.0.99)
```

Comment:
I don't believe this is correct syntax for RPM. Did you test this with an old version of OMI installed for the RPM case?

See [SCXCore](https://github.com/Microsoft/SCXcore/blob/38d4949059b0e21ee072d9f83353acd2d7268753/installer/datafiles/Linux_RPM.data#L6) for more information.

Reply:
Seems to be a bit confusing, for DPKG the version is 1.3.0.99 and for RPM the version is 1.3.0-99. Thanks for pointing that out. PR updated.